content-cache/docs/changelog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2024-12-16
+
+### **Added**
+
+- Support for active healthchecks using LUA scripts. Confofigured through healthcheck-interval and healthcheck-path.
```

Comment:
I think there is a typo `Confofigured`.

Reply:
Fixed in fd24ec3d7a8d66a860b21810fe8aac018415c9d8